_episodes/09-pull-requests.md

```diff
@@ -133,17 +211,18 @@ $ git commit -m "Added file on france"
 {: .bash}
 
 ~~~
-[master 79a312a] Added file on france
+[addFrance 79a312a] Added file on france
  1 file changed, 2 insertions(+), 2 deletions(-)
 ~~~
 {: .output}
 
 Now we can push those changes to our forked version of the repo.
 Since we don't have permissions to the upstream repo or we want to suggest these changed be
```

Comment:
Beautiful!  Like your proposed change a lot.  Will edit that and make another commit now.